docs/source/install/kernel_install.rst

```diff
@@ -3,9 +3,13 @@
 Installing the IPython kernel
 =============================
 
-The Jupyter Notebook and other frontends automatically ensure that the IPython kernel is available.
+The Jupyter Notebook and other frontends automatically ensure that the IPython kernel is available
+for whatever version of Python you used to install the Jupyter Notebook.
+
 However, if you want to use a kernel with a different version of Python, or in a virtualenv or conda environment,
-you'll need to install that manually.
+you'll need to install that manually. Note that this will **overwrite** the existing IPython kernel. If you want
```

Comment:
This is where it gets complicated. If you're setting it up to make the other one of Python 2/3 available to your notebook server, it *won't* overwrite your existing kernel, because it automatically calls them `python3` and `python2`. We've sort of assumed that that's the most common use case, which is why this section is before the explanation of giving them different names below.